autograd/nn.py

```diff
@@ -344,7 +370,7 @@ def extract_windows(x, kernel_size, stride, padding_mode="valid"):
 
     Returns:
         tuple: (windows, output_shape)
-            - windows: Stacked tensor of shape (num_windows, batch_size, channels, kernel_size, kernel_size)
+            - windows: Stacked tensor of shape (H_out, W_out, batch_size, channels, kernel_size, kernel_size)
```

Comment:
The function description wasn't updated, even though the windows were already in this shape. This somewhat misled the model not training problem.